Makefile

```diff
@@ -45,14 +43,10 @@ test-image: ## Creates a nodejs test image
 		-t ${TEST_IMAGE_NAME} \
 		./test-image
 
-push-test-image-sbx: test-image ## Pushes the nodejs test image to the sandbox registry
+push-test-image: test-image ## Pushes the nodejs test image to the non-prod registry for CI
```

Comment:
you shouldn't be pushing here manually, when developing the chart you should really use the sandbox registry